crates/sui-core/src/consensus_adapter.rs

```diff
@@ -299,9 +298,8 @@ impl ConsensusAdapter {
                     .sequencing_estimated_latency
                     .set(latency.as_millis() as i64);
                 let delay_step = latency * 3 / 2;
-                const MAX_DELAY_MUL: usize = 10;
```

Comment:
Do we want to increase instead of removing completely? Ex set to 20/30 ? Or we see absolutely no value on it?

Reply:
I think the value of retrying after the first few decreases rapidly. And a non-gradual ramp up generates instability to the system. If we regularly find > 10 resubmissions are needed, we can remove this. But at that point, probably there are more important latency issues that can be fixed.